central/serviceaccount/search/searcher_impl.go

```diff
@@ -7,15 +7,22 @@ import (
 	v1 "github.com/stackrox/rox/generated/api/v1"
 	"github.com/stackrox/rox/generated/storage"
 	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/search/paginated"
 )
 
+const whenUnlimited = 1000
```

Comment:
I was curious if this pattern is used elsewhere, and I found that it is.

Once with a value of 10, and in another place with a value of 100.

https://github.com/search?q=repo%3Astackrox%2Fstackrox%20whenUnlimited&type=code